client/dashboard/emails/choose-email-solution/index.tsx

```diff
@@ -1,25 +1,251 @@
+			poweredBy: {
+				logo: poweredByTitanLogo,
+				text: __( 'Powered by Titan' ),
+			},
+			product: titanProduct,
+			hasFreeTrial: hasTitanFreeTrial,
+			available: isTitanAvailable,
+			route: addProfessionalEmailRoute.fullPath,
+		},
+		{
+			logo: <img src={ GoogleLogo } alt="" />,
+			name: __( 'Google Workspace' ),
+			action: __( 'Get Google Workspace' ),
+			description: __(
+				'Business email with Gmail. Includes other collaboration and productivity tools from Google.'
+			),
+			features: [
+				__( 'Send and receive from your custom domain' ),
+				__( '30GB storage' ),
+				__( 'Email, calendar, and contacts' ),
+				__( 'Video calls, docs, spreadsheets, and more' ),
+				__( 'Real-time collaboration' ),
+				__( 'Store and share files in the cloud' ),
+				__( '24/7 support via email' ),
+			],
+			product: googleProduct,
+			available: isGoogleAvailable,
+			route: addGoogleMailboxRoute.fullPath,
+		},
+	];
 
 	return (
-		<PageLayout header={ <PageHeader /> } size="small">
-			<Text size={ 18 } as="h1">
-				{ __( 'Choose email solution' ) }
-			</Text>
-			<div style={ { marginTop: 12 } }>
-				<Text>
-					{ __( 'Domain:' ) } { domain }
-				</Text>
+		<PageLayout
+			header={ <PageHeader /> }
+			size="small"
+			notices={
+				<>
+					{ ! canAddEmail && (
+						<EmailNonDomainOwnerNotice
+							selectedSite={ site }
+							domain={ domain }
+							source="email-comparison"
+						/>
+					) }
+					{ hasEmailForwards( domain ) && <ExistingForwardsNotice /> }
+				</>
+			}
+		>
+			{ /* Billing interval selector */ }
+			<div className="billing-interval-selector">
+				<ToggleGroupControl
+					__next40pxDefaultSize
+					__nextHasNoMarginBottom
+					isBlock
+					label={ __( 'Billing interval' ) }
+					hideLabelFromVision
+					value={ billingInterval }
+					onChange={ ( newBillingInterval ) =>
+						setBillingInterval( newBillingInterval as IntervalLength )
+					}
+				>
+					<ToggleGroupControlOption label={ __( 'Monthly' ) } value="monthly" />
+					<ToggleGroupControlOption
+						/* translators: %d is the annual savings percentage. */
+						label={ sprintf( __( 'Annually (save up to %d%%)' ), bestAnnualSavings ) }
+						value="annually"
+					/>
+				</ToggleGroupControl>
+			</div>
+
+			{ /* Split card for providers */ }
+			<div className="email-providers">
+				{ providers.map( ( provider, providerIndex ) => (
+					<VStack className="email-provider" key={ `provider-${ providerIndex }` } spacing={ 4 }>
+						<Icon icon={ provider.logo } size={ 30 } className="email-provider-logo" />
+						<Text as="h2" size={ 28 } lineHeight="40px" className="email-provider-name">
+							{ provider.name }
+						</Text>
+						<Text>{ provider.description }</Text>
+						<VStack
+							spacing={ 2 }
+							justify="flex-start"
+							style={ { minHeight: hasTitanFreeTrial ? '96px' : '76px' } }
+						>
+							{ provider.available ? (
+								<>
+									<HStack expanded={ false } alignment="bottomLeft">
```

Comment:
What does `expanded={ false }` do? I've tried toggling the value and I didn't see any difference.

Reply:
Probably a leftover when I was polishing the layout. It's indeed no longer needed, so I removed it in https://github.com/Automattic/wp-calypso/pull/106555/commits/3726e8152c2abc1005cf4f5b2f8271376e35cef7